raiden-ts/src/transfers/epics.ts

```diff
@@ -476,125 +574,87 @@ export const transferGenerateAndSignEnvelopeMessageEpic = (
  *
  * @param action$ - Observable of transferSigned actions
  * @param state$ - Observable of RaidenStates
+ * @param latest$ - RaidenEpicDeps latest
  * @returns Observable of messageSend actions
  */
 export const transferSignedRetryMessageEpic = (
```

Comment:
`transferSignedRetryMessageEpic$`?

Reply:
I am not sure if the epics should also follow the same naming scheme (@andrevmatos could what do you think about that). My basic reasoning at least for the epics it is that since we are using redux-observable it is kind of implied that the Epics return an Observable/stream of actions.

Reply:
Seems like an odd convention then 🤷‍♂